puppet/zulip/manifests/profile/standalone.pp

```diff
@@ -14,5 +14,4 @@
   include zulip::profile::memcached
   include zulip::profile::rabbitmq
   include zulip::localhost_camo
-  include zulip::static_asset_compiler
```

Comment:
I think there's theoretically a use case of wanting to do `tools/i18n/sync-translations` on a production system, but if you want to do that, you can just `apt install gettext`; it's not a workflow we recommend anywhere.